boards/arm/disco_l475_iot1/disco_l475_iot1.dts

```diff
@@ -121,9 +121,13 @@
 	status = "okay";
 };
 
+&spi3_miso_pc11{ slew-rate = "very-high-speed";};
```

Comment:
```suggestion
&spi3_miso_pc11 {
        slew-rate = "very-high-speed";
};
```

Reply:
@martinjaeger this is actually coming from https://github.com/zephyrproject-rtos/zephyr/pull/29231 on which current PR is based on (besides, if you don't like this notation, you'll find more to hint on https://github.com/zephyrproject-rtos/zephyr/pull/29267).
Based on this can you reconsider your review on current PR and continue on #29231 or #29267 ?

Reply:
Sure, will have a look at the other PRs aswell.